src/cloud-element-templates/util/propertyUtil.js

```diff
@@ -729,24 +729,38 @@ export function unsetProperty(commandStack, element, property) {
     });
   }
 
-  // bpmn:Message#property
+  // bpmn:Message#zeebe:subscription#property
   if (type === MESSAGE_ZEEBE_SUBSCRIPTION_PROPERTY_TYPE) {
     const subscription = findZeebeSubscription(businessObject);
 
     if (!subscription) {
       return;
     }
 
-    commands.push({
-      cmd: 'element.updateModdleProperties',
-      context: {
-        ...context,
-        moddleElement: subscription,
-        properties: {
-          [ binding.name ]: undefined
+    // remove subscription of none properties are left
+    if (noPropertiesLeft(subscription, binding.name)) {
```

Comment:
The alternative is to handle this via a behavior, cf. [camunda-bpmn-js-behaviors](https://github.com/camunda/camunda-bpmn-js-behaviors).

Reply:
Hmm perhaps it makes more sense. I declined this option initially because I thought it's an element-templates-only problem, but the cleanup is broken also in non-templated messages.